// Copyright 2020 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package feedwire;

option optimize_for = LITE_RUNTIME;

message ChromeClientInfo {
  optional string session_id = 3;
  optional bool start_surface = 4;
}
